### PR TITLE
details for bom-phase1.md

### PR DESCRIPTION
**Pull request by mgheorghe**
component and vendor selection proposal

### CODE REVIEW

File: lab/bom-phase1.md
Comment (pdp2shirts) [inline lab/bom-phase1.md:1]: At line 19 the PDU line. We are only getting a single circuit to start with so we only need a single PDU.
At lines 43 and line 67 aquaired should be acquired

for the Marvell details on the DPU - @Prasun Kapoor gave these part numbers -- Model: EV-CRB-CN106-A1-R1. Part: EV-CRB-CN106-A1-R1. We still have not received an approved server list from Marvell.

**Comment by pdp2shirts** [comment]
> @mgheorghe can you please explain what is the exact usage of those components from the picture:
> 
> * 3 (KVM)
> * 6 (Virt)
> * 7 (Storage)
> * 8 (Tgen)
> * 9 (Test SW)
@glimchb 
The KVM will be used to provide remote Keyboard, Video, and Mouse to the server hosts (virt server, DPU hosts, etc)
The Virt is a host with a VMware ESXi Hypervisor on it that will be used to host necessary lab services like DHCP, DNS, Lab firewall, etc. --since we are not using a Lab services provider we will have to setup our own lab services, but the $ tradeoff for doing so was deemed worth it.
The Storage is another host used for general lab storage necessary to run the lab services and may have other uses as well.
The Tgen is a traffic generator host.
and the Test SW is a switch to which the test/DPUs and DPU hosts will be connected. It will be a higher throughput switch to handle the DPU network speeds, whereas the TS at top is where mgmt ports, KVM, virt host, etc will be connected

**Comment by mgheorghe** [comment]
> 

updated the lab diagram to match the items in the table

KVM is to be able to grab "physical" control of video signal and mouse and keyboard in a unified way for all devices in lab.
similar functionality can be provided via iDrac, iLo .... but this works for any device that accepts a monitor and keyboard and mouse irrespective if they have a bmc like module or not.

**Comment by mgheorghe** [comment]
@glimchb there was a concern regarding the number of cores on the server hosting the DPU, number of cores is still 2x16 cores

**Comment by pdp2shirts** [comment]
I approve, but this cannot be merged until one of the maintainers approves

**Review by glimchb — APPROVED** [review]

**Pull request closed without merging**
